test/unit/sessions/screen-recording-session-handler.test.ts

```diff
@@ -153,3 +150,55 @@ describe('updateOutgoingMedia', () => {
     }
   });
 });
+
+describe('updateScreenRecordingMetadatas', () => {
+  it('should replace trackIds with mid', async () => {
+
+
+    const metas = [
+      {
+        trackId: 'trackId123'
+      },
+      {
+        trackId: 'trackId456'
+      }
+    ];
+
+    const transceivers = [
+      {
+        mid: '3',
+        sender: {
+          track: { id: metas[0].trackId }
+        }
+      },
+      {
+        mid: '5',
+        sender: {
+          track: null
+        }
+      },
+      {
+        mid: '7',
+        sender: {
+          track: { id: metas[1].trackId }
+        }
+      },
+    ];
+
+    const session = {
+      pc: {
+        getTransceivers: jest.fn().mockReturnValue(transceivers)
+      }
+    };
+
+    jest.spyOn(utils, 'requestApi').mockResolvedValue(null);
+
+    await handler['updateScreenRecordingMetadatas'](session as any, metas as any);
+
+    expect(metas[0].trackId).toBe('3');
+    expect((metas[0] as any)._trackId).toBe('trackId123');
+
+    expect(metas[1].trackId).toBe('7');
+    expect((metas[1] as any)._trackId).toBe('trackId456');
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9447336)
```suggestion
    expect((metas[1] as unknown)._trackId).toBe('trackId456');
```